Server/Services/Implementations/SmtpService.cs

```diff
@@ -15,16 +15,26 @@ public SmtpService(IConfiguration configuration)
         }
 
 
-        public async Task SendRegisterConfirmationEmailAsync(string registerDtoEmail, string confirmationCode)
+        public async Task SendRegisterConfirmationEmailAsync(string email, string confirmationCode)
         {
-            var smtpClient = new SmtpClient(_smtpConfigSection["Host"])
+            var smtpClient = CreateSmtpClient();
+            await smtpClient.SendMailAsync(_smtpConfigSection["Email"], email, "QUANDA - Complete register", $"https://localhost:44358/api/accounts/confirm-email/{confirmationCode}");
+        }
+
+        public async Task SendPasswordRecoveryEmailAsync(string email, string recoveryJwt, int idUser)
+        {
+            var smtpClient = CreateSmtpClient();
+            await smtpClient.SendMailAsync(_smtpConfigSection["Email"], email, "QUANDA - Recover password", $"https://localhost:44358/api/accounts/recover-password/{idUser}/{recoveryJwt}");
```

Comment:
Suggestion - put link to new global variable so that it could be changed lately and to make code more readable